src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_payments.html.twig

```diff
@@ -1,11 +1,15 @@
-<div class="ui segment" id="payment-state">
-    {% include '@SyliusAdmin/Order/Label/PaymentState/' ~ order.paymentState ~ '.html.twig' with { 'value': 'sylius.ui.' ~ order.paymentState, 'attached': true } %}
+<div class="ui segment center aligned" id="payment-state">
     {% if order.hasPayments %}
+    {% include '@SyliusAdmin/Order/Label/PaymentState/' ~ order.paymentState ~ '.html.twig' with { 'value': 'sylius.ui.' ~ order.paymentState, 'attached': true } %}
         <h3 class="ui dividing header">{{ 'sylius.ui.payments'|trans }}</h3>
         <div class="ui relaxed divided list" id="sylius-payments">
             {% for payment in order.payments %}
                 {% include '@SyliusAdmin/Order/Show/_payment.html.twig' %}
             {% endfor %}
         </div>
+    {% else %}
+        <span class="ui grey large label" id="no-payments">
```

Comment:
Can we extract this `id` to object above? Also, can we change it to the data attribute?

Reply:
IMO no, because it is `id` of this specified element.